include/nuttx/uorb.h

```diff
@@ -678,4 +736,201 @@ struct sensor_ioctl_s
   char data[1];                /* The argument buf of ioctl */
 };
 
+/* This structure describes the information of the sensor device and
+ * requires the manufacturer to implement the device info function.
+ */
+
+struct sensor_device_info_s
```

Comment:
why add

Reply:
It`s a mistake while processing merge conflict because of depends, fixed.